.github/workflows/nightly.yml

```diff
@@ -32,21 +30,26 @@ jobs:
           file: gradle.properties
           property: minecraft_version
 
-      - name: Set up JDK
-        uses: actions/setup-java@v2
+      - name: Set up JDK 21
```

Comment:
The branch will use it's own copy of the nightly.yml config - so no need to add conditionals for the JDK version; it will always be correct for the branch that is being built (as far as I remember anyway)

Reply:
i _think_ whatever the set "main" branch is that's what it would use as the source for workflow

Reply:
it'd be worth testing before committing to added complexity :) but fairly sure from what I've seen in the past is branches running an older verify.yml from it's source - rather than from the "main" branch

Reply:
checked actions it seems to be taking it from the target branch